src/hooks/useVerifiableCredential.ts

```diff
@@ -82,10 +82,12 @@ export function useVerifiableCredential() {
           }
         }
       } catch (error: any) {
-        setSignError({
-          type: VCSignErrorType.SignError,
-          message: error.message,
-        });
+        if (error?.code !== 4001) {
+          setSignError({
+            type: VCSignErrorType.SignError,
+            message: error.message,
+          });
+        }
       }
       return null;
```

Comment:
What happens when null is sent. e.g error code is 4001?
Will AccountTEEndorsement handle the null scenario?

Reply:
4001 means user deined to sign in metamask

null case has handled in UI, will just keep stay in the modal, as deined to sign is not error or fail

Reply:
In that case it's ok